.github/workflows/kind.yml

```diff
@@ -81,12 +81,22 @@ jobs:
       run: |
         export GOPATH=$(go env GOPATH)
         make e2e-dependencies
-        make e2e-test
+        make e2e-test-coverage
 
-    - name: SonarCloud Analysis
-      if: ${{ github.event_name == 'push' }}
+    - name: Test Coverage and Report Generation
+      if: ${{ matrix.kind == 'latest' }}
       run: |
-        echo "SonarCloud not implemented."
+        make test-coverage | tee report_unit.json
+        make gosec-install
+        make gosec-scan
+        cat gosec.json
+
+    # - name: SonarCloud Analysis
```

Comment:
Why is this commented out?

Reply:
I didn't want my tests to duplicate the scans on Prow while they were still enabled. I'll remove this commit once they're gone in Prow.